### ISSUE TITLE
Release on PyPi

### ISSUE THREAD

**Issue by Tigge** [Tigge/openant#9]

**Comment by aspiers** [comment]
Any progress on this?  I'm no Python module expert, but it looks to me like all the hardware on the code (`setup.py` etc.) is already done, so isn't this pretty much just a matter of doing the upload to PyPi?  Would be great to be able to install via `pip`, and this would also unblock Tigge/antfs-cli#123.

**Comment by Tigge** [comment]
Yes everything should be ready to be uploaded to PyPi actually. I've been holding of in hopes of getting some more bug fixes in but perhaps it doesn't really matter what comes first here. I'll look into this!

**Comment by Echelon9** [comment]
I'd be in favor of a PyPi release too.

**Comment by purpl3F0x** [comment]
Hi, would you like to port the project to pillow and release to pypi ?

**Comment by tuna-f1sh** [comment]
https://pypi.org/project/openant
